corehq/ex-submodules/pillowtop/management/commands/create_checkpoints_for_merged_pillows.py

```diff
@@ -100,8 +98,15 @@ def handle(self, **options):
             print("Checkpoint creation cancelled")
             return
 
+        path = options['reorg_file_path']
```

Comment:
since we need to just add support for custom ones, should this just need the file to have the custom pillows and just append them to PILLOW_REORG_MAPPING?
Don't see why there is a need to mention all pillows if you need custom ones.

Reply:
Some envs might not have some pillows and some have extra ones, so it's hard to do it without a full spec. It also allows consolidating pillows few at a time, instead of all at once.